pkgs/os-specific/linux/switcheroo-control/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, ninja, meson, fetchFromGitLab, systemd, libgudev, pkgconfig, glib, python3Packages, gobject-introspection }:
+python3Packages.buildPythonApplication rec {
+  pname = "switcheroo-control";
+  version = "2.3";
+  src = fetchFromGitLab {
+    hash = "sha256-1Pze2TJ9mggfcpiLFwJ7/9WhsdJx4G3GoA7+Z47shuc=";
+    domain = "gitlab.freedesktop.org";
+    owner = "hadess";
+    repo = pname;
+    rev = version;
+  };
+  nativeBuildInputs = [
+    ninja meson pkgconfig glib
```

Comment:
In new expressions, `pkg-config` should be used instead of the `pkgconfig` alias.